deploy/templates/FhirSynapseLinkTemplate.json

```diff
@@ -15,11 +15,15 @@
                 "description": "The fhir server endpoint you wish to get data."
             }
         },
-        "managedFhirServerName": {
-            "type": "string",
-            "defaultValue": "",
+        "authentication": {
+            "type": "bool",
+            "defaultValue": false,
+            "allowedValues": [
+            true,
+            false
+            ],
             "metadata": {
-                "description": "The name of the Azure API for FHIR resource. Used for managed identity authentication. Leave it empty if you are using an OSS server instance."
+                "description": "Whether fhir server enable authencation checkness, if true will set up managed identity authentication. set false when using an OSS public fhir server instance."
```

Comment:
maybe: `Whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance.`

Reply:
Update in updated FhirSynapsePipelineTemplate.json